src/vue/backend/src/App.vue

```diff
@@ -1,12 +1,81 @@
 <template>
-  <idx-block className="app">
-    <idx-block className="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
+    <idx-block className="app {'nav-is-expanded': expanded}">
```

Comment:
the `className` prop will render out as a string and not eval the object you've passed.
You need something like this (note the colon before the attribute):
`:className="{ 'app': true, 'nav-is-expanded': expanded }"`